pkg/provisioner/fixture/fixture.go

```diff
@@ -14,6 +14,7 @@ import (
 var log = logf.Log.WithName("fixture")
 var deprovisionRequeueDelay = time.Second * 10
 var provisionRequeueDelay = time.Second * 10
+var requeueDelay = time.Second * 10
```

Comment:
The other delay values have more descriptive names. How is this new one used?

Reply:
Sorry, I miss this delay value from previous version of prototype :) I removed it.